src/components/ImageGrid/ImageCell.vue

```diff
@@ -42,6 +52,9 @@
 
 <script>
 import VLicense from '~/components/License/VLicense.vue'
+import VModal from '~/components/VModal/VModal.vue'
+import VImageResult from '~/components/VImageResult.vue'
```

Comment:
```suggestion
```

Removing this and  VImageResult from the component object below strangely fixes the `Unknown custom element` issue.